### ISSUE TITLE
Style preview via radio button mouseover doesn't work for all styles

### ISSUE THREAD

**Issue by olafgrabienski** [Graham-72/beautytipsX#3]
On the configuration page `admin/config/content/beautytips` there is the section "Styling options" to choose a default style. You can mouse over the radio buttons to see a preview of the different styles. The previews show up for the most but not all styles on my test site:

- plain => ok
- netflix => ok
- facebook => ok (but looks the same as *hulu*)
- transparent => ok
- big-green => ok
- **google-maps => doesn't show up**
- hulu => ok (but looks the same as *facebook*)
- **Teststyle => doesn't show up** (custom style from `admin/config/content/beautytips/custom-styles`)

**Comment by Graham-72** [comment]
I found that the preset styles for the so-called 'google-maps' style needed some values for 'cssStyles' in order to get it to work. I also found that some of values in other styles needed to be changed in order to give a satisfactory final result when using the 'beautytips' style on site pages/nodes, although they looked satisfactory on the admin page.

I am not sure why these preset styles are given the names 'netflix' 'facebook' etc and I think it might be better to use neutral names with no claim to match certain proprietary systems.

I am also not sure that it is a good idea for these preset values to be set by code in beautytips.module. I think it might be better in Backdrop to move them to the config file.

**Comment by olafgrabienski** [comment]
> I am not sure why these preset styles are given the names 'netflix' 'facebook' etc and I think it might be better to use neutral names (...)

Yes, I agree. The names might have been useful as a reference when the Drupal module was created, but I guess the styles aren't up to date any more.

Apart of that: will test the changes soon.

**Comment by olafgrabienski** [comment]
I've tested the changes, and now the *google maps* style is visible as a preview, nice!

Unfortunately, the custom style on my test site (built on `admin/config/content/beautytips/custom-styles`) continues to not be visible as a preview. It works on a content item.

**Comment by Graham-72** [comment]
Yes, I can see where the problem is but this week I am on holiday in
Scotland, away from my PC.

**Comment by olafgrabienski** [comment]
Ah, no hurry! Have a nice holiday!
